### PR TITLE
S6930: Add .NET ITs

### PR DESCRIPTION
**Pull request by antonioaversa**
Part of #8926

### CODE REVIEW

**Review by mary-georgiou-sonarsource — COMMENTED** [review]
Let's try to delete as many files as possible from the project, that are not needed for the analysis, to reduce the noise.

**Comment by antonioaversa** [comment]
> Let's try to delete as many files as possible from the project, that are not needed for the analysis, to reduce the noise.
@mary-georgiou-sonarsource Done, I could delete:
- the entire `wwwroot` content for all ASP.NET Core projects: the app still starts with a blank page and a 404, but that should be enough for the ITs
- `Content` (mostly `css` and related) and `Scripts` for the ASP.NET 4.x project: the app starts and display a full-text page, that should be more than enough for the ITs

File: analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/AspNet4x/Web.Release.config [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
```
Comment (mary-georgiou-sonarsource) [inline analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/AspNet4x/Web.Release.config:1]: Maybe we can also keep only ony Web.config file?

**Review by mary-georgiou-sonarsource — APPROVED** [review]
LGTM - one minor comment.

File: analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/AspNet4x/Web.Release.config (hunk 1 above)
Comment (antonioaversa) [reply analyzers/its/Projects/ManuallyAddedNoncompliantIssues.CS/AspNet4x/Web.Release.config:1]: Yes, the app still compiles and starts. I have removed both `Web.Release.config` and `Web.Debug.config`.

**Pull request merged**